schema/workflow.json

```diff
@@ -558,28 +558,19 @@
             "string",
             "object"
           ],
-          "description": "If string type, an expression which selects parts of the states data output to become the data (payload) of the event referenced by 'triggerEventRef'. If object type, a custom object to become the data (payload) of the event referenced by 'triggerEventRef'."
+          "description": "If string type, an expression which selects parts of the states data output to become the data (payload) of the event referenced by 'produceEventRef'. If object type, a custom object to become the data (payload) of the event referenced by 'produceEventRef'."
         },
         "contextAttributes": {
           "type": "object",
           "description": "Add additional extension context attributes to the produced event",
           "additionalProperties": {
             "type": "string"
           }
-        },
-        "invoke": {
-          "type": "string",
-          "enum": [
-            "sync",
-            "async"
-          ],
-          "description": "Specifies if the function should be invoked sync or async. Default is sync.",
-          "default": "sync"
```

Comment:
We will remove this attribute, but in the spec we don't have any reference to it?

Reply:
i think it was there by cut/paste. for eventRef it should assume async (event-driven) invocation